agent/agent.go

```diff
@@ -3110,7 +3110,7 @@ func (a *Agent) addCheckLocked(check *structs.HealthCheck, chkType *structs.Chec
 	return nil
 }
 
-func (a *Agent) addCheck(check *structs.HealthCheck, chkType *structs.CheckType, service *structs.NodeService, persist bool, token string, source configSource) error {
+func (a *Agent) addCheck(check *structs.HealthCheck, chkType *structs.CheckType, service *structs.NodeService, token string, source configSource) error {
```

Comment:
Looks like maybe this `persist` param was made unnecessary in 2aac4d5168cc761ca58f2500e65ec1874cb9f168 (#5012)